controllers/specializer-operator/Makefile

```
@@ -16,15 +16,24 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+
+# Only enable bash if it is in the PATH: 
+# this check is needed to be compatible with the busybox container used by PROW presubmit tests (that doesn't contain bash)
+HAVE_BASH ?= $(shell bash --version > /dev/null 2>&1; echo $$?)
```

Another option could be to use `command -v`
```suggestion
HAVE_BASH ?= $(shell command -v bash > /dev/null 2>&1; echo $$?)
```

---

I accepted your change, will push it soon